service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -156,7 +156,7 @@ internal class RevurderingServiceImpl(
             return KunneIkkeOppretteRevurdering.FantIkkeAktørId.left()
         }
 
-        // TODO ai 25.02.2021 - Oppgaven skal egentligen ikke opprettes her. Den burde egentligen komma utifra melding av endring, som skal føres til revurdering.
+        // Oppgaven skal egentligen ikke opprettes her. Den burde egentligen komma utifra melding av endring, som skal føres til revurdering.
```

Comment:
Dette er kanskje enda mer aktuelt nå? Nå vil vi få 2 oppgaver for det som gjelder personhendelser.

Reply:
Den er ikke helt dekket ännu, finns många andra cases som skal føre till revurdering så tänker att jag bara låter kommentaren vara kvar men tar bort TODO:n. ☺️